scout/adapter/mongo/case.py

```diff
@@ -779,7 +779,7 @@ def update_case(self, case_obj, keep_date=False):
                     "smn_tsv": case_obj.get("smn_tsv"),
                     "status": case_obj.get("status"),
                     "sv_rank_model_version": case_obj.get("sv_rank_model_version"),
-                    "track": case_obj.get("track"),
+                    "track": case_obj.get("track", "rare"),
```

Comment:
:+1:

Reply:
This works: 

![image](https://user-images.githubusercontent.com/28093618/115013637-9750ff80-9eb1-11eb-9715-1e088f00067e.png)

I'm going to merge now!